src/detection/scripts/data_extent.py

```diff
@@ -0,0 +1,121 @@
+# flake8: noqa
+
+folders = ['20170607_024038_101d', '20170607_024039_101d',
```

Comment:
This file seems more like data than code to me. It seems like it should be a json file that's included as part of the dataset. 

However, I'm not sure why this information needs to be listed explicitly. It seems like the extents could be inferred from the data files, and then directory names can be discovered using the `os` module in Python.